app/models/participant.rb

```diff
@@ -0,0 +1,7 @@
+class Participant < ApplicationRecord
+  #TODO: Add relationships
+  # has_many :paperwork
+  # has_many :case_notes
+  # enum status {r0: 0, r1: 1, r2: 2, studio: 3}
```

Comment:
This enum can be uncommented! I also think you're missing a colon after status - it should be `status:`. 

Also let me know when you have time and we can go over how to use `rails console` to check that the models/enums are working correctly :)